src/vuegraf/vuegraf.py

```diff
@@ -110,7 +110,7 @@ def createDataPoint(account, chanName, watts, timestamp, detailed):
         }
     return dataPoint
 
-def extractDataPoints(device, usageDataPoints):
+def extractDataPoints(device, usageDataPoints, startTime=None, stopTime=None):
```

Comment:
This is looking pretty good. I think if you change these parameter names to `historyStartTime` and `historyStopTime` it will avoid confusion.